### DISCUSSION TITLE
签到推送注册设置

### DISCUSSION THREAD

**Discussion by cn0068** [qd-today/qd#226] [Q&A] [answered]
企业微信，用;来分割，企业ID;应用ID;应用密钥;自定推送图片(非必需)
如下正确吗？
wwa6aaxxxxxxxxxc:1000001:xxxxxxxxxxxxxxxxxfc6xxx24TRDr-bFMgdujC9kxxx
这样设置后提示注册成功，测试已推送，但企业微信收不到消息。

**Comment by a76yyyy** [comment, marked as answer]
用英文分号分割，不是冒号

**Reply by cn0068** [reply]
已成功接受到微信推送的消息。谢谢。另外，我是用群晖的docker安装的，请问如何升级签到框架？

**Reply by a76yyyy** [reply]
如果已经把数据库映射到容器外部，那么建议用watchtower等工具升级
